merkle_tree_wasm/package.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "@anonklub/merkle-tree",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "compile": "source ./scripts/compile-web.sh"
```

```suggestion
    "compile": "wasm-pack build . --target bundler --out-dir wasm-pkg --out-name index --scope anonklub"
```

or better, if we want to check if `wasm-build` is installed first
```suggestion
    "check-wasm-pack": "command -v wasm-pack >/dev/null 2>&1 || echo \"wasm-pack is not installed\"\n",
    "compile": "pnpm run check-wasm-pack && wasm-pack build . --target bundler --out-dir wasm-pkg --out-name index --scope anonklub"
```

---

That's a pretty much better version.